modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * understanding_openshift_gitops/about-redhat-openshift-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-openshift-go-common-terms_{context}"]
+= Glossary of common terms for OpenShift {gitops-shortname}
```

Comment:
_OpenShift {gitops-shortname}_ doesn't seem to be a common term, only one time in the repo that I can find. Should this be just _{gitops-shortname}_